Src/MoneyFox.Windows.Business/BackgroundTaskManager.cs

```diff
@@ -2,28 +2,68 @@
 using System.Linq;
 using Windows.ApplicationModel.Background;
 using MoneyFox.Foundation.Interfaces;
+using System.Collections.ObjectModel;
+using MoneyFox.Foundation.DataModels;
+using MoneyFox.Foundation.Interfaces.Repositories;
+using MoneyFox.Business.Manager;
 
 namespace MoneyFox.Windows.Business
 {
     public class BackgroundTaskManager : IBackgroundTaskManager
     {
-        private const string TASK_NAMESPACE = "MoneyFox.Windows.Tasks";
+        private readonly ISettingsManager settingsManager;
+        private readonly ICategoryRepository categoryRepository;
 
+        private const string TASK_NAMESPACE = "MoneyFox.Windows.Tasks";
         private const string CLEAR_PAYMENT_TASK = "ClearPaymentTask";
         private const string RECURRING_PAYMENT_TASK = "RecurringPaymentTask";
         private const string SYNC_BACKUP_TASK = "SyncBackupTask";
 
+        public BackgroundTaskManager(ICategoryRepository categoryRepository, ISettingsManager settingsManager)
+        {
+            this.categoryRepository = categoryRepository;
+            this.settingsManager = settingsManager;
+        }
+
+        public bool firstUse
+        {
+            get { return settingsManager.firstUse; }
+        }
+
         public async void StartBackgroundTask()
         {
             var requestAccess = await BackgroundExecutionManager.RequestAccessAsync();
 
             if (requestAccess == BackgroundAccessStatus.DeniedByUser 
                 || requestAccess == BackgroundAccessStatus.DeniedBySystemPolicy) return;
 
-
             RegisterClearPaymentTask();
             RegisterRecurringPaymentTask();
             RegisterSyncBackupTask();
+
+            if (firstUse)
+            {
+                addDefaults(categoryRepository);
+                settingsManager.firstUse = false;
+            }            
+        }
+
+        private static void addDefaults(ICategoryRepository categoryRepository)
+        {   
+                CategoryViewModel Food = new CategoryViewModel();
+                CategoryViewModel Rent = new CategoryViewModel();
+                CategoryViewModel Insurance = new CategoryViewModel();
+                CategoryViewModel Entertainment = new CategoryViewModel();
+
+                Food.Name = "Food";
```

Comment:
Please use here translations from the Ressource.resx in the Foundation, so we can translate it.